### ISSUE TITLE
[iOS] In-Context Progress

### ISSUE THREAD

**Issue by nizmaylova** [openedx/openedx-app-ios#122]
User should see in the course outline what assignments have been completed. Completed subsections and modules should be highlighted in green. Also, the “Completed” mark should be displayed after  task has been completed.

![Image](https://github.com/openedx/openedx-app-ios/assets/121940663/ba5a88cc-d95b-474a-b12c-1ea25ca2f9c0)

**Comment by moiz994** [comment]
@volodymyr-chekyrta can you please confirm if this is already implemented in the app or not? 

I've tested it and a tick icon appears on some content while on others it does not.

![Screenshot_20231110-115926_OpenEdX](https://github.com/openedx/openedx-app-ios/assets/109789173/e7158400-99ce-4f58-a3c0-2274e2ee5563)
![Screenshot_20231110-115934_OpenEdX](https://github.com/openedx/openedx-app-ios/assets/109789173/4a2bf9bb-3fe4-4186-8a04-eeaaeaff3034)

**Comment by edschema** [comment]
@IvanStepanok agreed on a comment from Slack conversation: the completed checkmark should be enough; would not highlight completed subsections in green.
